packages/components/src/placeholder/style.scss

```diff
@@ -176,9 +176,18 @@
 	border-radius: inherit;
 	display: flex;
 	box-shadow: none;
-	background: none;
 	min-width: 100px;
 
+	// Blur the background so layered dashed placeholders are still visually separate.
+	// We also provide a semitransparent background so as to allow duotones to sheen through.
+	backdrop-filter: blur(100px);
```

Comment:
Is -webkit-backdrop-filter: blur(100px); also needed before backdrop-filter: blur(100px); for Safari? @jasmussen

Reply:
The included autoprefixer should take care of that, which is why I didn't include it. But excellent sanity check, thank you!

<img width="1392" alt="safari" src="https://user-images.githubusercontent.com/1204802/185439284-ca5509df-dc71-4dfb-9753-419a158d4c1f.png">